Feature/identidade visual (#68)
UnBArqDsw2021-1/2021.1_G04_Cardeal: docs/modelagem/identidadeVisual.md

<h6 align = "center">Figura 1 - Logo, versão 1.0.</h6>

![Logo](../desenhoSoftwareBase/img/IdentidadeVisual/logo.png)

<h6 align = "center">Fonte: Site <a href="https://www.figma.com/proto/DKWzd4WjiCNoDUUINY1U4v/Identidade-Visual-Cardeal?node-id=6%3A6" target="_blank">Figma</a></h6></br>
 
### 4. Tipografia

<p align = "justify"> Tipografia é o formato e o arranjo das palavras, define a composição visual dos textos. A tipografia escolhida para o projeto foi divida em duas partes, a prima parte representa a fonte que compõe a Logo, a fonte escolhida foi <a href="https://fonts.google.com/specimen/Sora?query=sora" target="_blank">Sora</a>, e a segunda parte representa as fontes usadas no website, escolhidas duas fontes <a href="https://fonts.google.com/specimen/Sora?query=sora" target="_blank">Sora</a> e <a href="https://fonts.google.com/specimen/Lora?query=lora#standard-styles" target="_blank">Lora</a>.</p>

### 4.1. Fonte - Sora

<p align = "justify">A fonte <a href="https://fonts.google.com/specimen/Sora?query=sora" target="_blank">Sora</a> foi escolhida para a Logo e para o uso no website.

> Mais informações sobre a fonte e licença de uso dinponibilizados pelo site <a href="https://fonts.google.com/specimen/Sora?query=Jonathan+Barnbrook" target="_blank">fonts.google</a></p>

Reviewer: ```suggestion
Mais informações sobre a fonte e licença de uso são disponibilizadas pelo site <a href="https://fonts.google.com/specimen/Sora?query=Jonathan+Barnbrook" target="_blank">fonts.google</a>.</p>
```

<h6 align = "center">Figura 2 - Representação da fonte Sora, versão 1.0.</h6>

![Sora](../desenhoSoftwareBase/img/IdentidadeVisual/sora.png)

<h6 align = "center">Fonte: Site <a href="https://www.figma.com/proto/DKWzd4WjiCNoDUUINY1U4v/Identidade-Visual-Cardeal?node-id=50%3A5" target="_blank">Figma</a></h6></br>

### 4.1. Fonte - Lora

<p align = "justify">A fonte <a href="https://fonts.google.com/specimen/Lora?query=Cyreal" target="_blank">Lora</a> foi escolhida para o uso no website.

> Mais informações sobre a fonte e licença de uso dinponibilizados pelo site <a href="https://fonts.google.com/specimen/Sora?query=Jonathan+Barnbrook" target="_blank">fonts.google</a></p>

Reviewer: ```suggestion
Mais informações sobre a fonte e licença de uso são disponibilizadas pelo site <a href="https://fonts.google.com/specimen/Sora?query=Jonathan+Barnbrook" target="_blank">fonts.google</a>.</p>
```

<h6 align = "center">Figura 3 - Representação da fonte Lora, versão 1.0.</h6>

![Lora](../desenhoSoftwareBase/img/IdentidadeVisual/lora.png)

<h6 align = "center">Fonte: Site <a href="https://www.figma.com/proto/DKWzd4WjiCNoDUUINY1U4v/Identidade-Visual-Cardeal?node-id=50%3A86" target="_blank">Figma</a></h6></br>

### 5. Cores

> <p align = "justify"> A cor possui sua influência na psicologia humana, e as escolhas das core representam o tipo de mensagem que deseja transmitir. A escolha das cores foi realizada nas cores que compõe o símbolo do projeto e buscando cores análogas ao resultado para possuir maior váriações de cores, para essa consulta foi realizada por meio do site <a href="https://coolors.co/" target="_blank">Coolors</a>. Escolhidos apenas 4 cores para compor o projeto, para possuir um visual mais limpo.</p><br/>

Reviewer: ```suggestion
<p align = "justify"> A cor possui sua influência na psicologia humana, e as escolhas das cores representam o tipo de mensagem que deseja transmitir. A escolha foi realizada baseando-se nas cores que compõe o símbolo do projeto e buscando cores análogas ao resultado para possuir maiores váriações de cores. Essa consulta foi realizada por meio do site <a href="https://coolors.co/" target="_blank">Coolors</a>. Foram escolhidas apenas 4 cores para compor o projeto, para possuir um visual mais limpo.</p><br/>
```

<h6 align = "center">Figura 4 - Cores principais, versão 1.0.</h6>

[![Cor-Principal](../desenhoSoftwareBase/img/IdentidadeVisual/corPrincipal.png)](../desenhoSoftwareBase/img/IdentidadeVisual/corPrincipal.png)

<h6 align = "center">Fonte: Site <a href="https://coolors.co/f14843-79797b-ffffff-020106-3f3d56" target="_blank">Coolors</a></h6></br>

| **Cor** | **Nome** | **RGB** | **CMYK** | **WEB** | 
| :---: | :---: | :---: | :---: | :---: | 
| ![Cor-Tart-Orange](../desenhoSoftwareBase/img/IdentidadeVisual/red.png) | Tart Orange | 241, 72, 67 | 0, 70, 72, 5 | #F14843 |
| ![Cor-Sonic-Silver](../desenhoSoftwareBase/img/IdentidadeVisual/grey.png) | Sonic Silver | 121, 121, 123 | 2, 2, 0, 52 | #79797B |
| ![Cor-White](../desenhoSoftwareBase/img/IdentidadeVisual/white.png) | White | 255, 255, 255 | 0, 0, 0, 0  | #FFFFFF |
| ![Cor-Black](../desenhoSoftwareBase/img/IdentidadeVisual/black.png) | Black | 2, 1, 6 | 67, 83, 0, 98 | #020106 |
| ![Cor-Black](../desenhoSoftwareBase/img/IdentidadeVisual/purple.png)  | Independence | 63, 61, 86 | 27, 29, 0, 66 | #3f3d56 |

### 5.2. Cores Secundárias

> <p align = "justify"> As cores secundárias representam as tonalidades diferentes das cores principais, para usar tons diferentes no projeto.</p>

Reviewer: ```suggestion
<p align = "justify"> As cores secundárias representam as diferentes tonalidades das cores principais, incluindo assim diferentes tons ao projeto.</p>
```

* [awwwards: The awards of design, creativity and innovation on the internet](https://www.awwwards.com/), acessado dia 18 de agosto de 2021. 

* [Coolors: Create the perfect palette or get inspired by thousands of beautiful color schemes.](https://coolors.co/), acessado dia 18 de agosto de 2021.

* [Fonts Google: Font Lora, designed by Cyreal](https://fonts.google.com/specimen/Lora?query=lora#standard-styles), acessado dia 18 de agosto de 2021.

* [Fonts Google: Font Sora, designed by Jonathan Barnbrook, Julián Moncada](https://fonts.google.com/specimen/Sora?query=sora), acessado dia 18 de agosto de 2021.

* [github: Arquitetura-desenho - A monitoria](https://2019-2-arquitetura-desenho.github.io/wiki/dinamica_seminario_II/identidade_visual/), acessado dia 18 de agosto de 2021.

* [Figma: the collaborative interface design tool.](https://www.figma.com), acessado dia 18 de agosto de 2021.

## 8. Versionamento 

| Data | Versão | Descrição | Autor(es) | 
| :----: | :----: | :----: | :----: | 
| 18/08/2021 | 1.0 | Criação do documento | [Tomás Veloso](https://github.com/tomasvelos0) | 

> | 18/08/2021 | 1.1    | Debate e escolha da identidade visual       | [Todos os membros](https://unbarqdsw2021-1.github.io/2021.1_G04_Cardeal/)  |

Reviewer: ```suggestion
| 18/08/2021 | 1.1    | Debate e escolha da identidade visual       | [Todos os membros](https://unbarqdsw2021-1.github.io/2021.1_G04_Cardeal/)  |
| 20/08/2021 | 1.2    | Revisão do documento | Douglas Castro e Gustavo Duarte |
```

Aqui adicionei ao versionamento a nossa revisão.
